ProjectName/.husky/pre-commit

```diff
@@ -1,5 +1,6 @@
 #!/usr/bin/env sh
 . "$(dirname -- "$0")/_/husky.sh"
 
-npx tsc
-npx lint-staged
+yarn tsc
+# yarn prettier:fix
```

Comment:
Please, do not leave commented parts.
Basically you either need this one, so leave it uncommented. 
Or remove completely. Do not be scared. You always would find what you need in the documentation.